firebase-inappmessaging-display/src/main/java/com/google/firebase/inappmessaging/display/FirebaseInAppMessagingDisplay.java

```diff
@@ -185,22 +183,18 @@ public void clearFiamListener() {
   public void onActivityStarted(final Activity activity) {
```

Comment:
just to confirm - there isn't any additional cleanup we want to take advantage of on other lifecycle events, ex: onActivityCreated vs started or in onActivityDestroyed?

Reply:
I think those already do the correct cleanup as far as I know. May not be enough but I think it's correct in the scope of this change